client/src/components/aboutUs.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import '../stylings/aboutUs.css';
+
+function AboutUs() {
+  return (
+    <div className='pageDiv'>
```

Comment:
Usually we want to avoid the name of the component inside the className or id. It's usually seen as clutter. Often something like "container" is used to highlight that a tag is serving as a container. Personally, I tend to use general names like "about-us".

Also, names like "page", "sections", and "text' are generic and can be confused with tags in other components. We should try to make names specific to the component. E.g., "about-us-text". Also, if a tag doesn't appear more than one in the file, we should use id instead of className. The latter is for grouping multiple elements under the same styling